replies.go

```diff
@@ -140,7 +145,7 @@ var replies = []Reply{
 		excludeRoles: []Role{Donator},
 	},
 	{
-		pattern: `schematics?`,
+		pattern: `schematics`,
```

Comment:
Please add back the `?`. In regex syntax that means the preceding character (`s` in this case) is optional, so this pattern matches **both** `schematic` and `schematics`